content/docs/dvclive/api-reference/live/end.md

```diff
@@ -0,0 +1,40 @@
+# Live.end()
+
+Signals that the current experiment has ended.
+
+```py
+def end():
+```
+
+## Usage
+
+`Live.end()` gets automatically called when exiting the context manager:
+
+```py
+from dvclive import Live
+
+with Live() as live:
+    pass
+# live.end() has been called at this point
+```
+
+It is also called when the training ends for each of the supported
+[ML Frameworks](/doc/dvclive/api-reference/ml-frameworks)
```

Comment:
Not a blocker: include an example framework code block similar to the one above?

Reply:
It may make more sense to put all of the `save_dvc_exp` behavior under one `live` method.